src/utils/v8utils.h

```diff
@@ -20,18 +20,22 @@
 #include <nan.h>
 #include <string>
 #include <vector>
+#include <codecvt>
 
 namespace etcher {
 namespace v8utils {
-std::vector<std::string> GetArguments(v8::Local<v8::Array> arguments);
+std::vector<std::wstring> GetArguments(v8::Local<v8::Array> arguments);
 }  // namespace v8utils
 }  // namespace etcher
 
 #define YIELD_ERROR(CALLBACK, ERROR)                                           \
   {                                                                            \
-    v8::Local<v8::Value> argv[1] = {Nan::Error((ERROR))};                      \
-    Nan::MakeCallback(Nan::GetCurrentContext()->Global(), (CALLBACK), 1,       \
-                      argv);                                                   \
+    const uint16_t *message = (const uint16_t *)(ERROR).c_str();               \
```

Comment:
This is already a `wchar_t`. The `NewFromTwoByte` takes an array of `uint16_t` (https://v8.paulfryzel.com/docs/master/classv8_1_1_string.html#a6b3efe5df1016e4f9f485e79b0acdf5b), so I'm basically casting it for that purpose.